src/main/java/de/retest/recheck/ignore/CompoundFilter.java

```diff
@@ -13,6 +13,10 @@ public CompoundFilter( final List<Filter> filters ) {
 		this.filters = filters;
 	}
 
+	public List<Filter> getFilters() {
```

Comment:
You just need this method in a test? I think, that you could just use the Filter methods to compare [this ](https://github.com/retest/recheck.cli/pull/62/files#diff-f4a532528223635b5a54a9dadb6877ceR86). If you really need this method, make it immutable `Collections.immutableList`, maybe even within the constructor.